test/unit/xiopak.unit.cpp

```diff
@@ -71,3 +82,57 @@ TEST(xiopak, file_operations)
     EXPECT_FALSE(xfExist(file1));
 
 }
+
+TEST(xiopak, path_functions) {
+
+    // Set up
+#define S(x) #x
+#define S_(x) S(x)
```

Comment:
The stringizing trick is fine but now the too-simple symbol `S(x)` exists everywhere. Remember what happened when we had a `#define D something`?

Reply:
I see. I'll give it a more readable/unique name.